src/components/back-to-top/_back-to-top.scss

```diff
@@ -0,0 +1,36 @@
+.ons-back-to-top {
+  > a > span {
```

Comment:
Instead of using `a` here and in the other places you could use `&__link`. Makes the code a bit more descriptive and otherwise that class doesn't seem to be being used.
You could even add a class to the span something like `ons-back-to-top__description`